### ISSUE TITLE
SDDM fails to read /etc/sddm.conf as well as /etc/sddm.conf.d/XXX

### ISSUE THREAD

**Issue by jw-rav** [sddm/sddm#1247]
To prevent powering down the machine by accident, I have put following contents to /etc/sddm.conf and to /etc/sddm.conf.d/01-my-config:

   [General]
   HaltCommand=/bin/false
   RebootCommand=/bin/false

Those files are set to mode 0644 and the directories are set to mode 0755, so there should be no permission problems.

But "sddm --example-config" still gives:

   [General]
   # Halt command
   HaltCommand=/usr/bin/systemctl poweroff
   
   # Input method module
   InputMethod=qtvirtualkeyboard
   
   # Comma-separated list of Linux namespaces for user session to enter
   Namespaces=
   
   # Initial NumLock state. Can be on, off or none.
   # If property is set to none, numlock won't be changed
   # NOTE: Currently ignored if autologin is enabled.
   Numlock=none
   
   # Reboot command
   RebootCommand=/usr/bin/systemctl reboot

This is on opensuse Leap.15.1, which contains sddm-0.18.0-lp151.3.3.1.x86_64

**Comment by jw-rav** [comment]
Sorry, I can't figure how to deactivate markup within the `code` section... still trying...

**Comment by Vogtinator** [comment]
`sddm --example-config` only dumps the built-in defaults, it does not read any configuration files.

**Comment by jw-rav** [comment]
OK, I see.

So how would I track down why sddm won't honor the HaltCommand and RebootCommand settings?

**Comment by Vogtinator** [comment]
It might depend on the theme which command is used. At least the built in themes and breeze use it correctly, so the buttons there should not do anything.

AFAICT your config should work.

**Comment by bengan** [comment]
> `sddm --example-config` only dumps the built-in defaults, it does not read any configuration files.

So why does "man sddm" say that it print the _current_ config to stdout?

       --example-config
              Print the complete current configuration to stdout

**Comment by Vogtinator** [comment]
Yeah, that's plain wrong. The behaviour was changed in d3c1e7574025fdb800aa804d6a7fe016d64c2080 but the man page never adjusted.

**Comment by felagund** [comment]
> So how would I track down why sddm won't honor the HaltCommand and RebootCommand settings?

I think that is more because these options not always work: https://github.com/sddm/sddm/issues/434

> So why does "man sddm" say that it print the _current_ config to stdout?

There is nothing about it in the manual - maybe it has been removed in the meantime? Instead the info is there when one runs `sddm -h`.

> Yeah, that's plain wrong. The behaviour was changed in [d3c1e75](https://github.com/sddm/sddm/commit/d3c1e7574025fdb800aa804d6a7fe016d64c2080) but the man page never adjusted.

It still says the incorrect thing (BTW it would have made sense to me to have both options - to print example config and to print actual config).
